images/scripts/degimpify.sh

```diff
@@ -12,9 +12,9 @@ shopt -s nullglob
 input_dirs="$*"
 
 if [ $# -eq 0 ]; then
-    input_dirs[0]="./EditButtons"
-    input_dirs[1]="./TranscriptionImages"
-    input_dirs[2]="./ControlButtons"
+    input_dirs[0]="${BASH_SOURCE[0]}/../../images/EditButtons"
+    input_dirs[1]="${BASH_SOURCE[0]}/../../images/TranscriptionImages"
+    input_dirs[2]="${BASH_SOURCE[0]}/../../images/ControlButtons"
```

Comment:
See lines 3-8.

Reply:
I tried this syntax on a few of my systems and it sometimes does not work.
Also, collapsed it to `$DIRNAME`

```suggestion
    DIRNAME=$(dirname "${BASH_SOURCE[0]}")
    input_dirs[0]="$DIRNAME/../../images/EditButtons"
    input_dirs[1]="$DIRNAME/../../images/TranscriptionImages"
    input_dirs[2]="$DIRNAME/../../images/ControlButtons"
```